routes/api/users.js

```diff
@@ -4,17 +4,20 @@ import {
   login,
   logout,
   currentUser,
+  updateAvatar,
 } from "../../controllers/user-controllers.js";
 import { auth } from "../../config/auth.js";
 
 const userRouter = express.Router();
 
-userRouter.post("/signup", signup);
+userRouter.post("/signup", auth, signup);
 
-userRouter.post("/login", login);
+userRouter.post("/login", auth, login);
 
-userRouter.post("/logout", logout);
+userRouter.post("/logout", auth, logout);
 
-userRouter.get("/cuttrent", currentUser, auth);
+userRouter.get("/current", currentUser, auth);
```

Comment:
Nie sprawdzałem tych kontrolerów ale 
auth -> middleware ? jeśli tak to powinien być przed currentUser (jeśli currentUser) to controler.
```suggestion
userRouter.get("/current", auth, currentUser);
```